### DISCUSSION TITLE
HELP FOR SECURITY/TECHNICAL AUDIT

### DISCUSSION THREAD

**Discussion by kcelestinomaria** [reach-sh/reach-lang#1375] [Q&A] [answered]
Hello Reach Team,

Humbly requesting if you can help us do a security audit of our smart contracts before we launch, just a technical audit to help us know beforehand potential bugs and know what to expect in outcomes after launch on mainnet, just to be safe. Tried reaching out to several other companies on Algorand for several days now but it is really hard to get help let alone a response, so I reverted back here because I have been building with your help and you know more about the tech. We would really appreciate a third eye into our work. Thanks again.

**Comment by jeapostrophe** [comment, marked as answer]
Howdy, please email me at jay@reach.sh and we'll talk about it!
